src/interface/src/app/plan/plan-summary/saved-scenarios/saved-scenarios.component.ts

```diff
@@ -24,13 +24,13 @@ export class SavedScenariosComponent implements OnInit {
     this planning area can also view all the scenarios within it.
   `;
 
+  highlightedId: string = '-1';
```

Comment:
Same comment here, this should probably be a number no? If not maybe we should use `null` when you don't have any selected